test/cpp/interop/xds_interop_client.cc

```diff
@@ -67,6 +67,8 @@ ABSL_FLAG(int32_t, stats_port, 50052,
           "Port to expose peer distribution stats service.");
 ABSL_FLAG(std::string, rpc, "UnaryCall",
           "a comma separated list of rpc methods.");
+ABSL_FLAG(int32_t, request_payload_size, 0, "request payload size in bytes.");
```

Comment:
should we clarify what's the contents of the payload be?

Reply:
seems fine to me, as the content isn't important as long as they get to the desired length. It's kind of documented here: https://github.com/grpc/grpc/blob/master/doc/interop-test-descriptions.md#large_unary

` body: 271828 bytes of zeros `

Reply:
Updated the comment

Reply:
I see. I'll give it another though.

Reply:
https://github.com/grpc/grpc/blob/master/doc/interop-test-descriptions.md#large_unary

OK, these specs are not for psm (xds) interop, but for the regular grpc interop tests. We don't even have `large_unary` test case.